packages/master-detail-layout/src/vaadin-master-detail-layout.js

```diff
@@ -550,29 +550,41 @@ class MasterDetailLayout extends SlotStylesMixin(ResizeMixin(ElementMixin(Themab
    * When passing null as the element, the current detail element is removed.
    *
    * If the browser does not support view transitions, the respective updates
-   * are applied immediately without starting a transition.
+   * are applied immediately without starting a transition. The transition can
+   * also be skipped using the `skipTransition` parameter.
    *
    * @param element the new detail element, or null to remove the current detail
+   * @param skipTransition whether to skip the transition
```

Comment:
Let's either update `.d.ts` accordingly or remove this method from typings since it's now protected.

Reply:
I'd prefer to treat this an internal, so I've removed it from the TS definitions for now.